users.js

```diff
@@ -1273,10 +1385,11 @@ User = (function () {
 		}
 
 		for (var ip in this.ips) {
-			lockedIps[ip] = userid;
+			lockedIps[ip] = [userid, expiration];
 		}
-		if (this.autoconfirmed) lockedUsers[this.autoconfirmed] = this.userid;
-		if (this.authenticated) lockedUsers[this.userid] = this.userid;
+		var entry = [this.userid, expiration];
```

Comment:
That's how it is in the current code, but now that I think about it, it would make more sense to track the <code>userid</code> like we do for bans.